### PR TITLE
Se modifica el README y se agrega index.js

### PR DESCRIPTION
**Pull request by EduardoContreras**
Comentario del PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
 Repositorio de prueba
 Archivo Readme
-Lorem Ipsu
\ No newline at end of file
+Lorem Ipsu
+
+
+
+## CAMBIOS EDU
+bla bla bla
```
Comment (mariofc) [inline README.md:9]: Qué significa este "bla bla bla"? xD

**Review by mariofc — COMMENTED** [review]
Comment example from Review changes

File: README.md (hunk 1 above)
Comment (EduardoContreras) [reply README.md:9]: hahahaha this is an example of file with changes

**Review by mariofc — APPROVED** [review]
Allow changes

**Pull request merged**